assignment7/allClients.js

```diff
@@ -0,0 +1,18 @@
+var express = require('express');
+var router = express.Router();
+var rp = require('request-promise')
+
+router.get('/',function(req, res){
+	var url = ('https://api.createsend.com/api/v3.1/clients.json');
+	rp.get(url,{
+		'auth' : {
+			'user' : 'fc96dc9ded698bbfbfb975409a864a42',
+			'pass' : 'x'
+		}
+	})
+		.then(function(response){
+		res.send(response);
```

Comment:
@abansal26 Please indent your code properly before creating the PR.

Reply:
Indentation improved. Commit ID - 2b780672efb014cb63fc8cff825b998ae65d0206